kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -810,6 +810,112 @@ p#site-title {
   height: 50%;
 }
 
+.kb-narr-outline {
+  --kb-narr-outline: 2px solid #cecece;
+  padding: 0.5em 0.5em 0.5em 0.25em;
+}
+
+.kb-narr-outline ul {
+  list-style: none;
+  padding: 0;
+  position: relative;
+}
+
+.kb-narr-outline ul ul {
+  list-style: none;
+  padding-left: 1.5em;
+  position: relative;
+}
+
+.kb-narr-outline li {
+  position: relative;
+}
+
+.kb-narr-outline ul ul li:not(:last-of-type)::before {
+  border-left: var(--kb-narr-outline);
+  content: "";
```

Comment:
is there somewhere that content is being added, hence this line is required to cancel it out?

Reply:
It (the ::before psuedoelement) needs to have content set to an empty string in order to render as a block. See https://www.w3.org/TR/CSS2/generate.html#content , it's a weird spec

Reply:
though, pretty standard practice